ceph/ceph_admin/bootstrap.py

```diff
@@ -92,3 +98,19 @@ def bootstrap(self: CephAdmProtocol, config: Dict) -> None:
         logger.error("Bootstrap error: %s", err.read().decode())
 
         self.distribute_cephadm_gen_pub_key()
+
+        # Verification of arguments
+
+        args = self.config.get("args", {})
+        logger.info(args)
+        for key, value in args.items():
+            func = self.get_method(key)
+            if not func:
+                continue
+            if not func(value):
+                raise AssertionError
+
+    def validate_fsid(self, fsid) -> bool:
+        out, err = self.shell(args=["ceph", "fsid"])
+        logger.info("Installed with custom fsid %s", fsid)
```

Comment:
Do we need to log this statement?

Reply:
@psathyan, Its not mandatory but I feel its good to print when something gets validated as messaging makes easier for us to understand when we go through the logs. Thoughts please.